pkg/visor/visor_test.go

```diff
@@ -1,94 +1,82 @@
-package visor
-
-import (
-	"encoding/json"
-	"io/ioutil"
-	"net/http"
-	"net/http/httptest"
-	"os"
-	"testing"
+// todo: tests here seems to be at very least of integration flavor: taking multiple
+// parts and checking if they work together
+// either remove them completely and rely on tests that setup infrastructure
+// or rewrite them to use mocks
```

Comment:
I agree with this, this sounds like an integration test.

Reply:
In this case, lets remove them and add them to the list of integration tests you are working on in `services` @alexadhy @i-hate-nicknames

Reply:
yes, good point. I will.

Reply:
@i-hate-nicknames please remove todo and remove the tests